core/src/main/java/com/capitalone/dashboard/model/RequestLog.java

```diff
@@ -0,0 +1,116 @@
+package com.capitalone.dashboard.model;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+
+@Document(collection = "requests")
+public class RequestLog extends BaseModel {
```

Comment:
maybe  just make a RequestLog toString() that spits out what u need for logging so when u have to log it then (and only then) will it generate that big string.. 